app/models/airline.rb

```diff
@@ -9,8 +9,9 @@ def self.search(query)
   end
 
   def average_rating
-    sum = self.reviews.all.inject(0) { |sum, review| sum += review.rating }
-    count = reviews.count || 1
+    return 0 if reviews.empty?
+    sum = reviews.all.inject(0) { |running_total, review| running_total += review.rating  }
```

Comment:
Useless assignment to variable - `running_total`. Use just operator `+`.<br>Line is too long. [91/80]<br>Unnecessary spacing detected.